app/components/request/configs.js

```diff
@@ -83,7 +126,7 @@ export default Component.extend(CanTriggerBuild, TriggerBuild, {
   },
 
   submit() {
-    this.set('processing', true);
+    this.set('submitting', true);
```

Comment:
Would it make sense to bind `submitting` to `this. submitBuildRequest.isRunning` instead of manual setting?

Reply:
yes, absolutely. in fact Emmanuel asked that same question on an earlier PR, and we've postponed it. but now that i'm making all changes to the final branch, there's no point in delaying things any further.

done: https://github.com/travis-ci/travis-web/commit/8d1e6dfbc237a6afe6d052ba44b5b3b109ccdabf